src/components/data-viewer/renderer.tsx

```diff
@@ -109,7 +113,7 @@ export default class DataViewer extends React.Component<Props, State> {
           </button>
           <div className="pagination-wrapper">{pagination}</div>
         </div>
-        <ErrorBoundary>{table}</ErrorBoundary>
+        {this.props.error ? <span className="error">Something is wrong. Check logs.</span> : table}
```

Comment:
Let's keep the error boundary.

Reply:
Instead of this error message, just open the error log automatically. The data viewer will never be useful when there are errors. For warnings, I propose https://github.com/vega/editor/issues/147